test/integration/compile-main.test.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const sass = require('node-sass');
+
+const BASE_PATH = path.join(__dirname, '../../');
+const MAIN_FILE = path.join(BASE_PATH, 'main.scss');
+
+async function compile({ file, includePaths }) {
+	return new Promise((resolve, reject) => {
+		sass.render({
+			file,
+			includePaths
+		}, function(err, result) {
+			if (err) {
+				return reject(err);
+			}
+			resolve(result);
+		});
+	});
+}
+
+describe('Compile scss file', () => {
+	test('main.scss', async () => {
+		const result = await compile({
+			file: MAIN_FILE,
+			includePaths: [
+				path.join(BASE_PATH, 'bower_components'),
```

Comment:
For reference, this was taken from https://github.com/Financial-Times/n-ui/blob/3fa35774540dc9cd956b4d6fdda1db04fe036e6e/scripts/build-sass.sh#L9-L10